server/server.js

```diff
@@ -13,6 +18,16 @@ app.get('/',(req, res) =>{
   res.status(200).send('hello world et tonton');
 })
 
-app.post('/',(req, res) =>{
-  res.status(200).send('hello world et tonton');
-})
\ No newline at end of file
+app.post('/boxes', (req, res) => {
+
+  const formData = req.body;
+  connection.query('INSERT INTO box SET ?', formData, (err, results) => {
+
+    if (err) {
+      console.log(err);
+      res.status(500).send("Erreur lors de la sauvegarde d'une box");
+    } else {
+      res.sendStatus(200).send(res);
```

Comment:
maybe you can find a better status for response ;)

https://en.m.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success